### PR TITLE
Add fslrobustfov

### LINKED ISSUE #1928: 🐛 Skull stripping or brain extraction with fsl BET does not properly remove the neck regions in some subjects.
### Describe the bug

When fsl BET is used for skull stripping or brain extraction, the neck regions from the anatomical image does not get removed for some subjects. 
As you can see in this example, the final output of fsl BET(red) includes the brain and the neck regions. 
image

In order correct for this issue, we need to use  fsl `robustfov`(https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/InitialProcessing) before using `BET`. 
For example, using `robustfov` in the case above, we get the following output image without the neck regions. 
image

### To reproduce

Only happens for some subjects. To reproduce this issue use fsl `BET` for brain extraction. The brain mask generated will also contain the neck regions in addition to the brain. 

### Preconfig

- [X] default
- [X] abcd-options
- [ ] anat-only
- [ ] blank
- [ ] ccs-options
- [ ] fmriprep-options
- [ ] fx-options
- [ ] monkey
- [ ] monkey-ABCD
- [ ] ndmg
- [ ] nhp-macaque
- [ ] preproc
- [ ] rbc-options
- [ ] rodent

### Custom pipeline configuration

_No response_

### Run command

_No response_

### Expected behavior

The output of fsl BET should  only include the brain without other regions of the head (neck, skull). 

### Acceptance criteria

fsl BET should extract only the brain and should be able to remove all the other additional regions of the head such as the skull and the neck for all subjects. 

### Screenshots

_No response_

### C-PAC version

_No response_

### Container platform

_No response_

### Docker and/or Singularity version(s)

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by tergeorge**
## Fixes

Fixes #1928  by @tergeorge 

## Description
The PR adds [fsl robustfov](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/InitialProcessing#:~:text=cropping%20to%20remove%20neck%20and%20lower%20head%3A%20robustfov%20(automatic)) before fsl BET is applied for skull stripping. This step is added in the anatomical preprocessing pipeline. 

## Technical details
fsl BET if applied alone does not remove the neck and other regions of the head for some subjects. fsl robustfov removes the neck and lower head regions. Addition of robustfov to the pipeline will improve the reliability of fsl BET for brain extraction.
## Tests
To test the PR:
1. Run CPAC with this PR on a subject with problematic skull stripping (using fsl BET). 
2. When BET is used alone the brain extraction does not give you a good output (the neck regions are also extracted), but when fsl robustfov is applied before BET, the neck regions are removed properly and only the brain is extracted. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the `develop` branch of the repository. 
- [x] My commit messages follow best practices.
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I updated the changelog.
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: CPAC/resources/configs/pipeline_config_blank.yml [hunk 1]
```diff
@@ -342,6 +342,7 @@ anatomical_preproc:
       monkey: Off
 
     FSL-BET:
+      Robustfov: On
```
Comment (shnizzedy) [inline CPAC/resources/configs/pipeline_config_blank.yml:345]: Do we want `Robustfov: On` in `blank` (and everything that imports `FROM: blank`), `Off` in `default` https://github.com/FCP-INDI/C-PAC/blob/d97e42c9f6a66e711746a9f72db8469e2a0ac7f6/CPAC/resources/configs/pipeline_config_default.yml#L412 (and everything that imports `FROM: default`)?

File: CPAC/resources/configs/pipeline_config_default.yml [hunk 2]
```diff
@@ -409,7 +409,7 @@ anatomical_preproc:
       monkey: False
 
     FSL-BET:
-
+      Robustfov : Off
```
Comment (shnizzedy) [inline CPAC/resources/configs/pipeline_config_default.yml:412]: Can you add a comment above this line in the default pipeline to explain what the option is and how to set it? The comment should propogate automatically to the other pipelines that include the key.

**Review by shnizzedy — CHANGES_REQUESTED** [review]

File: CPAC/resources/configs/pipeline_config_blank.yml (hunk 1 above)
Comment (sgiavasis) [reply CPAC/resources/configs/pipeline_config_blank.yml:345]: Yes, it should be default `Off`. Good catch!

Comment (birajstha) [reply CPAC/resources/configs/pipeline_config_blank.yml]: Thanks for catching this. I have changed to default `off`

File: CPAC/resources/configs/pipeline_config_default.yml (hunk 2 above)
Comment (birajstha) [reply CPAC/resources/configs/pipeline_config_default.yml:413]: Thanks. Added now.

File: CPAC/resources/configs/pipeline_config_blank.yml (hunk 1 above)
Comment (shnizzedy) [reply CPAC/resources/configs/pipeline_config_blank.yml]: Are there any preconfigs where `Robustfov: On` should be the default, or is this just a new option not included in any preconfigs?

**Review by shnizzedy — COMMENTED** [review]
> Are there any preconfigs where `Robustfov: On` should be the default, or is this just a new option not included in any preconfigs?

― https://github.com/FCP-INDI/C-PAC/pull/1933#discussion_r1507952466

**Comment by birajstha** [comment]
> > Are there any preconfigs where `Robustfov: On` should be the default, or is this just a new option not included in any preconfigs?
> 
> ― [#1933 (comment)](https://github.com/FCP-INDI/C-PAC/pull/1933#discussion_r1507952466)
Hi @shnizzedy , Sorry for late response here but, yes this is a new feature not included in any preconfigs.

**Review by shnizzedy — APPROVED** [review]
Looks good to me! I don't know if we want to turn this on in a test config, or leave it as a never-default option for now

**Pull request merged**